Sacaza Framework/Internal Interfaces/Collection Node Support/CollectionNodeSacaza.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CollectionNodeSacaza.swift
+//  ExpressionFramework
+//
+//  Created by Matt Harding on 24/11/2017.
+//  Copyright © 2017 Andriusstep. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import AsyncDisplayKit
+
+// MARK:- DataSource
+
+public protocol SacazaCollectionNodeDataSource {
+    
+    func numberOfSections(in collectionNode: ASCollectionNode) -> Int
+    func collectionNode(_ collectionNode: ASCollectionNode, numberOfItemsInSection section: Int) -> Int
+    func collectionNode(_ collectionNode: ASCollectionNode, nodeForItemAt indexPath: IndexPath) -> ASCellNode
+}
+
+// MARK:- Delegate
+
+public protocol SacazaCollectionNodeDelegate {
+    
+    func collectionNode(_ collectionNode: ASCollectionNode, didSelectItemAt indexPath: IndexPath)
+}
+
+// MARK:- class Declaration
+
+public final class CollectionNodeSacaza: NSObject {
```

Comment:
Not sure about sacaza suffixes and prefixes, because we can just use module namespace